# Array of waypoints
std_msgs/Header header
Wpnt[] wpnts
